components/eventing-controller/pkg/handlers/utils_test.go

```diff
@@ -42,16 +42,20 @@ func TestGetInternalView4Ev2(t *testing.T) {
 		}(),
 	}
 
-	defaultWebhookAuth := &types.WebhookAuth{
+	defaultWebhookAuth := types.WebhookAuth{
 		Type:         types.AuthTypeClientCredentials,
 		GrantType:    types.GrantTypeClientCredentials,
 		ClientID:     "clientID",
 		ClientSecret: "clientSecret",
 		TokenURL:     "tokenURL",
 	}
-
 	defaultNameMapper := NewBebSubscriptionNameMapper("my-shoot", 50)
 
+	bebSubEvents := eventingtesting.GetBEBEvents(types.Event{
+		Source: reconcilertesting.EventSource,
+		Type:   reconcilertesting.OrderCreatedEventType,
+	})
```

Comment:
Why we need extra func `GetBEBEvents` when we can do:

```golang
bebSubEvents := types.Events{types.Event{
    Source: reconcilertesting.EventSource,
    Type:   reconcilertesting.OrderCreatedEventType,
}}
```

Reply:
The intention was to reuse the method to reduce some code but it's just used only one hence removed.